Dockerfile

```diff
@@ -1,18 +1,34 @@
-FROM node:15 as builder
-LABEL author="chevdor@gmail.com"
+FROM docker.io/library/node:15 as builder
 
 WORKDIR /opt/builder
 
 COPY . .
-RUN curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh -s -- -y && \
-    . $HOME/.cargo/env && \
-    cargo install wasm-pack && \
-    yarn install && \
+
+RUN yarn install && \
     yarn build
 
 # ---------------------------------
 
-FROM node:15-alpine
+FROM docker.io/library/node:15-alpine
```

Comment:
Am I right in thinking `/library/` is equivalent to `/_/` on hub.docker.com?

Reply:
yes, `/_/` is the alias for `/library/`

Reply:
not sure about now, but earlier versions of buildah didn't have the docker registry configured by default and it was confusing. Buildah did not understand the docker-style shortened image addresses i.e. `node:15` or `parity/substrate-api-sidecar` and demanded a fqdn everywhere.
Then it softened and now buildah is able to deduct the image address (when you build manually with just `node:15` in `FROM` it will stop and ask where to take it from - QUAY or dockehub), but still it will throw an error on `push` if you built an image without the registry address in name.